pkg/autodiscovery/providers/kube_services.go

```diff
@@ -151,15 +156,23 @@ func parseServiceAnnotations(services []*v1.Service) ([]integration.Config, erro
 			continue
 		}
 		service_id := apiserver.EntityForService(svc)
-		c, errors := extractTemplatesFromMap(service_id, svc.Annotations, kubeServiceAnnotationPrefix)
+		svcConf, errors := extractTemplatesFromMap(service_id, svc.Annotations, kubeServiceAnnotationPrefix)
+		for _, err := range errors {
+			log.Errorf("Cannot parse template for service %s/%s: %s", svc.Namespace, svc.Name, err)
```

Comment:
It's probably worth logging which template (service/endpoints) failed to parse, wdyt?

Reply:
agree 👍